crates/nargo_cli/tests/compile_failure/dup_trait_implementation_3/src/main.nr

```diff
@@ -0,0 +1,19 @@
+trait Default {
+}
+
+struct MyStruct {
+}
+
```

Comment:
Do we really need `dup_trait_implementation_1`, 2, and 3 just to test duplicate traits? These look like they should all be covered by the same check to me, so there should only be one test.

Reply:
They're named a bit poorly, but `dup_trait_implementation_1` tests duplicate methods inside the same impl block, and `2` tests for duplicate impl blocks. `3` is similar to `2`, but it explicitly uses a typedef for the struct, I'd rather keep it in since we had a few troubles around them.